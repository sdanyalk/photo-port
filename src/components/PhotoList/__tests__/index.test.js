import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import PhotoList from '..';

afterEach(cleanup);

describe('PhotList component', () => {
    it('renders', () => {
        render(<PhotoList />);
    });

    it('matches snapshot DOM node structure', () => {
        const { asFragment } = render(<PhotoList />);

        expect(asFragment()).toMatchSnapshot();
    });
});
