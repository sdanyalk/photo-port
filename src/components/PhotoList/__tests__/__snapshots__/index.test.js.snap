// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotList component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div
    class="flex-row"
  />
</DocumentFragment>
`;
